CITATION.cff

```diff
@@ -0,0 +1,51 @@
+cff-version: 1.2.0
+title: Pyccel
+message: 'If you use this software, please cite the JOSS article.'
+type: software
+authors:
+  - given-names: Ahmed
+    family-names: Ratnani
+    orcid: 'https://orcid.org/0000-0001-9035-1231'
+    affiliation: 'UM6P, Morocco'
+  - given-names: Yaman
+    family-names: Güçlü
+    orcid: 'https://orcid.org/0000-0003-2619-5152'
+    affiliation: >-
+      NMPP division, Max-Planck-Institut für Plasmaphysik,
+      Garching bei München, Germany
+  - given-names: Emily
+    family-names: Bourne
+    orcid: 'https://orcid.org/0000-0002-3469-2338'
+    affiliation: >-
+      SCITAS, École polytechnique fédérale de Lausanne
+      (EPFL), Switzerland
+  - given-names: Said
+    family-names: Hadjout
+    orcid: 'https://orcid.org/0000-0003-3878-3146'
+  - given-names: Mohamed Jalal
+    family-names: Maaouni
+    affiliation: UM6P
```

Comment:
Maybe this could be expanded to "University Mohammed VI Polytechnic (UM6P), Morocco"?

Reply:
Fixed [aba19b9](https://github.com/pyccel/pyccel/pull/2222/commits/aba19b91882831c3b14b4969264e94e6b52b8eb9)